content/applications/hr/payroll.rst

```diff
@@ -457,22 +709,101 @@ advantage listed for a particular structure type is only available for that spec
    have meal vouchers. The opposite is true for someone hired under the :guilabel:`Intern` structure
    type. They would have meal vouchers available to them, not the use of a company car.
 
-To make a new advantage, click the :guilabel:`New` button, and enter the information in the fields.
-The required fields for an advantage are:
+To make a new benefit, click the :guilabel:`New` button, and enter the information in the fields on
+the blank benefits form.
 
-- :guilabel:`Name`: Enter the name for the advantage.
-- :guilabel:`Advantage Field`: Select from the drop-down menu what field in the payslip this
-  advantage appears under.
-- :guilabel:`Advantage Type`: Select from the drop-down menu what type of advantage the benefit is.
-  Select from :guilabel:`Monthly Benefit in Kind`, :guilabel:`Monthly Advantages in Net`,
-  :guilabel:`Monthly Advantages in Cash`, or :guilabel:`Yearly Advantages in Cash`.
-- :guilabel:`Salary Structure Type`: Select from the drop-down menu which salary structure type this
-  advantage applies to.
-- :guilabel:`Display Type`: Select from the drop-down menu how this advantage is displayed.
+The various fields for creating a benefit are as follows:
 
-.. image:: payroll/new-advantage.png
+General information section
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+- :guilabel:`Benefits`: enter the name for the benefit. This field is required.
+- :guilabel:`Benefit Field`: select from the drop-down menu the corresponding type of field in the
+  contract for this benefit.
```

Comment:
can this be reworded slightly? it reads a bit confusing in its current state.

Reply:
This was confusing to me, too! I reworded it.